src/lapis/app.lua

```diff
@@ -0,0 +1,13 @@
+local lapis = require("lapis")
+local db = require("lapis.db")
+
+local app = lapis.Application()
+
+app:get("/healthcheck", function()
+  local res = db.query("SELECT * FROM information_schema.tables WHERE table_schema = ? AND table_type = ?", "public", "BASE TABLE")
+  return {
+    json = {"OK!"}
+  }
+end)
```

Comment:
So, despite us using Lapis solely for migrations right now, it is a powerful web framework, has pretty good data modeling, etc. In the future, we could probably use it to build an API for the gamemode.

If you'd like to test this healthcheck, you can spin up Lapis.

```bash
$ docker-compose up lapis
zs_database_1 is up-to-date
Recreating zs_lapis_1 ... done
Attaching to zs_lapis_1
lapis_1      | nginx: [alert] lua_code_cache is off; this will hurt performance in /usr/local/src/nginx.conf.compiled:26
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: using the "epoll" event method
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: openresty/1.17.8.2
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: built by gcc 9.3.0 (Alpine 9.3.0)
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: OS: Linux 4.19.76-linuxkit
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: getrlimit(RLIMIT_NOFILE): 1048576:1048576
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: start worker processes
lapis_1      | 2020/09/04 02:17:12 [notice] 12#12: start worker process 14
```

```bash
$ http localhost:8080/healthcheck
HTTP/1.1 200 OK
Connection: keep-alive
Content-Type: application/json
Date: Fri, 04 Sep 2020 02:17:32 GMT
Server: openresty/1.17.8.2
Transfer-Encoding: chunked

[
    "OK!"
]
```